staging/src/k8s.io/legacy-cloud-providers/aws/aws.go

```diff
@@ -250,6 +250,9 @@ const (
 // The major consequence is that it is then not considered for AWS zone discovery for dynamic volume creation.
 var awsTagNameMasterRoles = sets.NewString("kubernetes.io/role/master", "k8s.io/role/master")
 
+// errorELBOnlySupportTCPLoadBalancer represents the error that only TCP LoadBalancer is supported for ELB
+var errorELBOnlySupportTCPLoadBalancer = fmt.Errorf("Only TCP LoadBalancer is supported for AWS ELB")
```

Comment:
No need to define a variable, unless you're exporting it

Reply:
I'm reusing `errorELBOnlySupportTCPLoadBalancer` in test: https://github.com/kubernetes/kubernetes/pull/80394/files#diff-38fe63834837be4a3b79a6260b5591d1R1404.